app/views/patron_requests/new.html.erb

```diff
@@ -31,15 +31,16 @@
 
             <tbody>
               <% f.object.items_in_location.each do |item| %>
+                <% patron_requestable = item.requestable?(current_user.patron) %>
                 <tr>
                   <td>
                     <label>
-                      <%= check_box_tag 'barcodes[]', value: item.barcode || item.id, class: 'form-check-input', disabled: cannot?(:request, item), data: { 'itemselector-target': 'items', 'itemselector-id-param': item.id, 'itemselector-available-param': item.available?, 'itemselector-label-param': item.callnumber, action: 'itemselector#change' }%>
+                      <%= check_box_tag 'barcodes[]', value: item.barcode || item.id, class: 'form-check-input', disabled: cannot?(:request, item), data: { 'itemselector-target': 'items', 'itemselector-id-param': item.id, 'itemselector-available-param': item.available?, 'itemselector-label-param': item.callnumber, action: 'itemselector#change' }, disabled: !patron_requestable %>
```

Comment:
There's already a ` disabled: cannot?(:request, item)` attribute on this check_box_tag.  I'm not sure what `patron_requestable` is, but I hope it's the same thing?

Reply:
Yeah. My bad I missed that.